volontulo/frontend/scss/customization/_bootflat-panel.scss

```diff
@@ -95,6 +95,10 @@ $panel-background-color:        $white !default;
 // Panel media
 //------------------------------------------------------
 .media {
+    > .panel-body,
```

Comment:
But the same code for these selectors are not repeated. This is the same as calling function in many places is not repeating code. We should have only one place that contain definitions for many selectors.